Build fails on MacOS

**Describe the bug**

The projecct is failing to build on MacOS;

```
[INFO] Reactor Summary for Horreum 0.12-SNAPSHOT:
[INFO] 
[INFO] Horreum ............................................ SUCCESS [  0.249 s]
[INFO] horreum-api ........................................ SUCCESS [  4.337 s]
[INFO] Horreum Client ..................................... SUCCESS [  0.770 s]
[INFO] Horreum infra - common ............................. SUCCESS [  0.725 s]
[INFO] Horreum Dev Services - Parent ...................... SUCCESS [  0.008 s]
[INFO] Horreum Dev Services - Runtime ..................... SUCCESS [  0.601 s]
[INFO] Horreum Dev Services - Deployment .................. SUCCESS [  0.803 s]
[INFO] Horreum Backend .................................... FAILURE [  2.740 s]
[INFO] Horreum Integration Tests .......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10.973 s
[INFO] Finished at: 2024-02-02T13:24:18Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.smallrye:jandex-maven-plugin:3.0.5:jandex (make-index) on project horreum-backend: Execution make-index of goal io.smallrye:jandex-maven-plugin:3.0.5:jandex failed: Cannot read the array length because "bytes" is null -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :horreum-backend

```

--- a/horreum-backend/pom.xml
+++ b/horreum-backend/pom.xml
@@ -264,28 +264,6 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>buildnumber-maven-plugin</artifactId>
             </plugin>
-            <plugin>
-                <groupId>io.smallrye</groupId>
-                <artifactId>jandex-maven-plugin</artifactId>
-                <version>3.0.5</version>
-                <executions>
-                    <execution>
-                        <id>make-index</id>
-                        <goals>
-                            <goal>jandex</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>15</source>
-                    <target>15</target>
-                </configuration>
-            </plugin>
-
         </plugins>
     </build>
 

